src/starkware/committee/committee/committee/committee.py

```diff
@@ -393,6 +398,13 @@ async def main():
         hash_func=pedersen_hash_func,
         availability_gateway=availability_gateway,
     )
+
+    metrics_config = config.get("METRICS", {})
+    if metrics_config.get("enabled", False):
+        port = 8000
```

Comment:
It's probably better to get the port number from the configuration. (8000 can be the default value.)
Also, it's not ideal to have this value set in 2 separate places (here and in Dockerfile).

Reply:
Any idea how I can avoid setting default port value in two places? For now, I left a comment about it in the default config.